requirements/base.in

```diff
@@ -1,4 +1,4 @@
 # Main requirements of your plugin application.
 -c constraints.txt
 
-Django
+eox-core
```

Comment:
Is this plugin installing properly in the devstack without breaking anything? @mariajgrimaldi

Reply:
Yes, everything seems to work just fine!